src/common/BO/loginBO.ts

```diff
@@ -0,0 +1,34 @@
+import testContext from '@utils/testContext';
```

Comment:
I think there is no need to 
For the next playwright version, i think that you will use storage stage and this file will be very different

Reply:
Have you some docs ?

Reply:
something like this https://playwright.dev/docs/auth#basic-shared-account-in-all-tests
but it will be done while migrating to Playwright/test

Reply:
I keep it for the moment and I will update before the release of 1.0.0.

Reply:
https://playwright.dev/docs/auth#basic-shared-account-in-all-tests